docs/conf.py

```diff
@@ -0,0 +1,183 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = u'Oppia'
+copyright = u'2019, Oppia'
+author = u'Audrey Tavares'
+
+# The short X.Y version
+version = u''
+# The full version, including alpha/beta/rc tags
+release = u'3.0'
+
+
+# -- General configuration ---------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#
+# needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    'sphinx.ext.autodoc',
+]
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+#
+# source_suffix = ['.rst', '.md']
```

Comment:
Delete unused code if you're replacing it; don't comment it out or check it in.

Reply:
Can I get more clarification on this? I don't really understand what I need to do. I don't remember changing anything on this section of the file (except for lines 22-24).

Reply:
Ah, sorry! On re-reading, this is my mistake, I didn't realize that this line was still part of the instructions. Please ignore this comment.